content/core-features/authentication.textile

```diff
@@ -468,33 +468,43 @@ h4. Examples
 
 ```[javascript]
-const ablyRest = new Ably.Rest({ key: '{{API_KEY}}' })
-const requestBody = {"targets": [ "clientId:client1@example.com" ]}                                                                                                                                                                                       
-                                                                                                                                                                                                                                                               
+const Ably = require('ably');
+
+const ablyRest = new Ably.Rest({ key: '{{API_KEY}}' });
+const requestBody = { targets: ['clientId:client1@example.com'] };
+
 ablyRest.request('post', '/keys/{{API_KEY_NAME}}/revokeTokens', null, requestBody, null,
-function(err, response) {                                                                                                        
-  if(err) {                                                                                                                                                                                                                                      
-    console.log('An error occurred; err = ' + err.toString());
-  } else {                                                                                                                                                                                                 
-    console.log('Success! status code was ' + response.statusCode);
-  }                                                                                                                                                                                            
-});                                                                                                                                                                                                                                                            
+
+  function (err, response) {
+    console.log(response);
+    if (!response.success) {
+      console.log('An error occurred; err = ' + response.errorMessage);
+    } else {
+      console.log('Success! status code was ' + response.statusCode);
+    }
+  }
+);
 ```
 
 A sample request to revoke a JWT based on @revocationKey@, using the REST client library SDK:
 
 ```[javascript]
+const Ably = require('ably');
+
 const ablyRest = new Ably.Rest({ key: '{{API_KEY}}' })
-const requestBody = {"targets": [ "revocationKey:client1@example.com" ]}                                                                                                                                                                                       
+const requestBody = { targets: ['revocationKey:client1@example.com'] }                                                                                                                                                                                       
```

Comment:
@tbedford I'm just wondering whether this should look more like a Key than just re-using the clientId from the preceding example. WDYT?

Reply:
![revoc](https://user-images.githubusercontent.com/18176755/184537291-0abd873d-3563-40f5-a443-55e87829f56b.png)

Reply:
Good point. `revocationKey` is an arbitrary string, but at the moment it's a bit confusing to use the same string in both examples.